oc_cdtapi/tests/test_BitbucketAPI.py

```
@@ -0,0 +1,263 @@
+
+
+class _Session(object):
+    """
+    Fake requests session
+    """
+
+    def __init__(self, handler):
+        self.handler = handler
+        self.auth = None
+
+    def get(self, req, params=None, **other):
```

Ruff (0.15.12)

[warning] Missing return type annotation for private function `get`

(ANN202)